python/pyspark/ml/torch/distributor.py

```diff
@@ -514,11 +514,26 @@ def _execute_command(
                 f"Command {cmd} failed with return code {task.returncode}. "
                 f"The {last_n_msg} included below: {task_output}"
             )
+    @staticmethod  
+    def _get_output_from_framework_wrapper(framework_wrapper: Optional[Callable], input_params: Dict, train_object: Union[Callable, str], run_pytorch_file_fn: Optional[Callable], *args, **kwargs) -> Optional[Any]:
+        if not framework_wrapper:
+            raise RuntimeError("In the _get_output_from_framework_wrapper function, found a framework wrapper that is none. I wonder why this is...")
```

Comment:
If there is ever a point where the framework_wrapper function isn't a Callable, we want this error to be thrown because it isn't supposed to happen. The reason we set this to an Optional[Callable] is because my linter complained a lot about it and how we can't assign something to framework_wrapper.

Reply:
How does the new one sound @lu-wang-dl?